vtex/faststore
# Chore | Search and Home view

## What's the purpose of this pull request?
This PR keeps the previous work on moving page-related code to a separate folder called `views`. With this, we get closer to Gatsby's doc on how to create custom page views.

This PR only moves code and does not introduce new features

## How to test it?
Enter in any one of these stores and make sure their home and plp are the same from their production counterparts

https://github.com/vtex-sites/btglobal.store/pull/570
https://github.com/vtex-sites/carrefourbrfood.store/pull/714
https://github.com/vtex-sites/marinbrasil.store/pull/511
https://github.com/vtex-sites/storecomponents.store/pull/937
